CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Opt-out compilation flag `winpty` to disable WinPTY support
 - Scrolling during selection when mouse is at top/bottom of window
 - Expanding existing selections using the right mouse button
+- Terminal search
```

Comment:
In general, I feel like being a bit more verbose here could be better.

Reply:
Any concrete suggestions?

Reply:
Well, it's a search for Vi mode, but it's a Terminal search on the other hand. And I'm not sure that `Scrollback buffer search` is a better name here, since it's not a scrollback buffer search, but generic search, since alt-screen doesn't have a scrollback.

Reply:
I've gone into a little more detail, hopefully that should make things a little more clear.